include/OpenColorIO/OpenColorIO.h

```diff
@@ -900,6 +910,8 @@ class OCIOEXPORT Config
     /// Returns the description attribute of a (display, view) pair.
     const char * getDisplayViewDescription(const char * display, const char * view) const noexcept;
 
```

Comment:
```
/**
 * \brief Determine if a display and view exist.
 *
 * This returns false if either the display or view doesn't exist. It works regardless
 * of whether the display or view are active, and it works regardless of whether the
 * view is display-defined or if the display has this as a shared view. It will only
 * check config-level shared views if dispName is null. It will not check config level
 * shared views if dispName is not null.
 */

```